### ISSUE TITLE
ARIMA_PLUS model erroring in audit_insert_template

### ISSUE THREAD

**Issue by lewisd-benefex** [kristeligt-dagblad/dbt_ml#47]
```
                array(
                    select as struct training_run, iteration, loss, eval_loss, learning_rate, duration_ms, array(select as struct null as centroid_id, cast(null as float64) as cluster_radius, null as cluster_size)
                    from ml.training_info(model `dataset`.`ml`.`ml_train_spend_forecast`)
                ) as training_info
```
            
10:10:40.077045 [error] [MainThread]: Database Error in model ml_train_spend_forecast (models/ml/ml_train_spend_forecast.sql)
10:10:40.077160 [error] [MainThread]:  Unrecognized name: loss at [14:63]

Due to deprecation [here](https://cloud.google.com/bigquery/docs/reference/standard-sql/bigqueryml-syntax-create-time-series) we should look to allow for both model types e.g.:

arima:
    training_info:
        - training_run
        - iteration
        - cast(null as float64) as loss
        - cast(null as float64) as eval_loss
        - cast(null as float64) as learning_rate
        - duration_ms
        - array(select as struct null as centroid_id, cast(null as float64) as cluster_radius, null as cluster_size)
    feature_info: *default_feature_info
arima_plus:
    training_info:
        - training_run
        - iteration
        - cast(null as float64) as loss
        - cast(null as float64) as eval_loss
        - cast(null as float64) as learning_rate
        - duration_ms
        - array(select as struct null as centroid_id, cast(null as float64) as cluster_radius, null as cluster_size)
    feature_info: *default_feature_info
